codbex-electra-ext-econt/api/SalesOrders.js

```diff
@@ -0,0 +1,39 @@
+const rs = require("http/rs");
+const http = require("codbex-electra/gen/api/utils/http");
+const configurations = require("core/configurations");
+
+const SHOP_SECRET_CFG_NAME = "ELECTRA_ECONT_SHOP_SECRET";
+const ECONT_DELIVERY_URL_CFG_NAME = "ELECTRA_ECONT_DELIVERY_URL";
+
+function getMandatoryCfg(configName) {
+	const configValue = configurations.get(configName);
+	if (configValue) {
```

Comment:
Could you invert this `if`:
```JS
if (!configValue) {
  throw new Error(...);
}
```

Reply:
I guess we can also add this functionality to the `configurations` API but we can do it some other time

Reply:
I like it this way